java/src/jmri/jmrix/loconet/ds64/Ds64TabbedPanel.java

```diff
@@ -2157,7 +2164,7 @@ public void initComponents() {
         localSensorType.setName("21"); // NOI18N
         localSensorType.addActionListener(basicConfigChangeActionListener);
 
-        factoryResetButton = new JToggleButton(Bundle.getMessage("ButtonResetToFactoryDefault"));
+        factoryResetButton = new JToggleButton(Bundle.getMessage("ButtonTextResetToFactoryDefault"));
```

Comment:
No objection to rename, but generally the keys for buttons(text) are just called ButtonWhat_it_says, which keeps them shorter both in the properties file as well as argument in the code.

Reply:
The rename came about because a single text resource was referred to by two different resource key names in two different places in the code.  I chose one.